.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,93 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: 🐞 Bug report
+about: Report a bug
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+/kind bug
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+<!--
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+Welcome! - We kindly ask you to:
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+  1. Fill out the issue template below 
+  2. Use Gitter if you have a question rather than a bug or feature request.
+    Gitter community channel is at: https://gitter.im/devfile/community
```

Comment:
Not a big deal, so no need to change, but for consistency we could make this second sentence in the imperative voice as well. So, something like, `See the Gitter community channel at:...`

Reply:
updated